src/styles/header.scss

```diff
@@ -0,0 +1,76 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+html {
+  font-size: 16px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
+  padding: 0 50px;
+  margin: 0 auto;
+
+  &__logo-link {
+    display: block;
+    height: 40px;
+    width: 40px;
+  }
+
+  &__menu {
+    display: flex;
+
+    &-link {
+      position: relative;
+      display: flex;
+      align-items: center;
+      height: 60px;
+      text-transform: uppercase;
+      font-size: 12px;
+      line-height: 16px;
+      font-weight: 500;
+
+      &--is-active {
+        color: #00acdc;
```

Comment:
For the `.header__menu-link--is-active` class, use a variable for the color if it's a primary or frequently used color in the project.